src/main/resources/static/seed.sql

```diff
@@ -17,15 +17,15 @@ sysdate() ),
 (1000004,  'Monica',  'Torres',  'monica@test.com',  '104',
 'Columbus Circle',  'New York',  'NY',  '10019',  true,  sysdate() );
 
-INSERT INTO userContact (userid, contactid) VALUES
-(1000000,  1234560),
-(1000000,  1234561),
-(1000000,  1234562),
-(1000000, 1000002),
-(1000002, 1234560),
-(1000002, 1234561),
-(1234560, 1000002),
-(1000002, 1000004);
+INSERT INTO userContact (userID, contactId, userContactId) VALUES
```

Comment:
Sorry about this.  We are back to the original table structure.  We will not have the userContactId.